docker/shared_steps/install_php_extensions.sh

```diff
@@ -28,6 +28,8 @@ php_install_extensions() {
   local already_installed_extensions=`php -m | egrep '^([A-Za-z_]+)$' | tr '[:upper:]' '[:lower:]'`
   local php_extensions_count=`echo $extensions | wc -w`
   local php_extensions_counter="1"
+  
+  mkdir '/usr/local/etc/php/conf.d/conf.d'
```

Comment:
I think we should move this to a separate function for the relevant extension.
Such as `php_install_ldap()`. This way we know, which extension needs the fix and keep the main function clean.
Please use the `-p` flag aswell.

Reply:
@vaighir why is this necessary with the new PHP versions? Or was it a bug before?

Reply:
I'm not sure why. In PHP 7.2.11 it's not necessary, but from 7.2.12 up the build fails with: 
`/usr/local/bin/docker-php-ext-enable: 108: /usr/local/bin/docker-php-ext-enable: cannot create /usr/local/etc/php/conf.d/conf.d/docker-php-ext-bcmath.ini: Directory nonexistent
BUILD FAILED!!!`

Reply:
would be interesting, what change caused this - as it looks like a typo `conf.d/conf.d`